docs/content/examples/tutorials/blog-posts.md

```diff
@@ -5,5 +5,4 @@ title: 'Tutorials | Blog Posts'
 This site lists blog entries that write about the project. If you blogged about `docker-mailserver` let us know so we can add it here!
 
 - [Installing docker-mailserver](https://lowtek.ca/roo/2021/installing-docker-mailserver/) by [@andrewlow](https://github.com/andrewlow)
-- [Simple mail-server with docker](https://tvi.al/simple-mail-server-with-docker/) by [@tomav](https://github.com/tomav)
```

Comment:
If removing instead of replacing with web archive link, might want to do the same on the main README for the project, one of the first links is "[Why this image was created](https://tvi.al/simple-mail-server-with-docker/)"

Reply:
I've removed the link from the README and added a sentence about the origin. If someone has a better phrasing, make a suggestion. Or should we remove the whole sentence?